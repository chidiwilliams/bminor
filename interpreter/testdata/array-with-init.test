a: array [5] integer = {1,2,3,4,5};
months: array [12] string = {
    "January","February","March","April","May","June","July",
    "August","September","October","November","December"
};

print a;
print "\n";
print months;
