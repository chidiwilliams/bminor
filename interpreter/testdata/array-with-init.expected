[1 2 3 4 5]
[January February March April May June July August September October November December]
